guides/Synthetic Data Generation.md

```
@@ -3,9 +3,9 @@
 
 ---
 
-# This guide as moved to [our docs](https://docs.getkiln.ai/docs/synthetic-data-generation). This page may be out of date.
+# This guide as moved to [our docs](https://docs.kiln.tech/docs/synthetic-data-generation). This page may be out of date.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)